bridge/mattermost/mattermost.go

```diff
@@ -323,6 +323,14 @@ func (m *Mattermost) MsgChannelThread(channelID, parentID, text string) (string,
 	props := make(map[string]interface{})
 	props["matterircd_"+m.mc.User.Id] = true
 
+	if parentID != "" {
```

Comment:
Any reason we need this check?
At the moment if a user gives an incorrect ID the user will receive a (private) error message on irc.

Reply:
Ah sure, removed.